gr-blocks/grc/blocks_trickling_throttle.block.yml

```diff
@@ -0,0 +1,81 @@
+id: blocks_trickling_throttle
+label: Throttle
+flags: [ throttle, python, cpp ]
+
+parameters:
+-   id: type
+    label: Type
+    dtype: enum
+    options: [complex, float, int, short, byte]
+    option_attributes:
+        size: [gr.sizeof_gr_complex, gr.sizeof_float, gr.sizeof_int, gr.sizeof_short,
+            gr.sizeof_char]
+    hide: part
+-   id: samples_per_second
+    label: Sample Rate
+    dtype: real
+    default: samp_rate
+-   id: vlen
+    label: Vector Length
+    dtype: int
+    default: '1'
+    hide: ${ 'part' if vlen == 1 else 'none' }
+-   id: ignoretag
+    label: Ignore rx_rate tag
+    dtype: bool
+    default: 'True'
+    hide: ${ ('part' if str(ignoretag) == 'True' else 'none') }
+-   id: duration_or_samples
+    label: Limited Quantity
+    dtype: enum
+    options: [True, False]
+    option_labels: ["wait duration", "#items per chunk"]
+-   id: maximum_items_per_chunk
```

Comment:
Can the two max's be collapsed into one? Hidden value are a pain (e.g., finding a hidden, invalid value in the asserts below). They're different types, so not sure what to do with that.